### ISSUE TITLE
Creating a form from a stencil results in an error

### ISSUE THREAD

**Issue by Mosnar** [verbb/formie#991]
### Describe the bug

When creating a form from a stencil, when I click "Next", the an error flash message is displayed. Upon inspection, the form was in fact created and I can navigate to it by clicking to the "Forms" tab. The templated form notifications are not present on the resulting form.

Here is the error from the formie.log file:

```
2022-07-15 11:33:54 [ERROR] Unable to save form “example”: SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'recipients' cannot be null
The SQL being executed was: INSERT INTO `craft_formie_notifications` (`formId`, `templateId`, `pdfTemplateId`, `name`, `enabled`, `subject`, `recipients`, `to`, `toConditions`, `cc`, `bcc`, `replyTo`, `replyToName`, `from`, `fromName`, `content`, `attachFiles`, `attachPdf`, `attachAssets`, `enableConditions`, `conditions`, `dateCreated`, `dateUpdated`, `uid`) VALUES (2667, NULL, NULL, 'Admin Notification', 1, 'A new submission was made on \"{formName}\"', NULL, '{systemEmail}', NULL, NULL, NULL, '{emailAddress}', NULL, '{emailAddress}', NULL, '[{\"type\":\"paragraph\",\"content\":[{\"type\":\"text\",\"text\":\"A user submission has been made on the \\\"\"},{\"type\":\"variableTag\",\"attrs\":{\"label\":\"Form Name\",\"value\":\"{formName}\"}},{\"type\":\"text\",\"text\":\"\\\" form on \"},{\"type\":\"variableTag\",\"attrs\":{\"label\":\"Site Name\",\"value\":\"{siteName}\"}},{\"type\":\"text\",\"text\":\" at \"},{\"type\":\"variableTag\",\"attrs\":{\"label\":\"Timestamp (yyyy-mm-dd hh:mm:ss)\",\"value\":\"{timestamp}\"}}]},{\"type\":\"paragraph\",\"content\":[{\"type\":\"text\",\"text\":\"Their submission details are:\"}]},{\"type\":\"paragraph\",\"content\":[{\"type\":\"variableTag\",\"attrs\":{\"label\":\"All Form Fields\",\"value\":\"{allFields}\"}}]}]', 1, NULL, NULL, 0, NULL, '2022-07-15 15:33:54', '2022-07-15 15:33:54', '1be1a174-e619-428d-a717-daa994042b34')
2022-07-15 11:33:54 [INFO] Request context:
$_GET = []

$_POST = [
    'action' => 'formie/forms/save'
    'redirect' => '3816a9fde47f2e652675f68b4aa71b4192814c7f636d0e439782f245a93aa2afformie/forms/edit/{id}'
    'CRAFT_CSRF_TOKEN' => 'YnCC_kG-qrI-wQNtHS_Q-nd5w3vvswWNLnAo1w4jO9M3IvHUbmEa7ro2YK2Xvfv6nPhLAuHe9ygUwTBB6aK1fk_dQ4mLca9mVbMg_mOqPSuJkRwYXiteJxAa9LioQ7Zj6Jk0gnV2s_WXquK612vUG9L7NBPcm3K63rmrbMTcOc8rpFY42ZEseblzXhQu2EJUo8JPAzVHu5UM2f_mc5dzC25f5MJFM-4_sOZB63RDR7E4E0mlXk6Hrl0nfrTGD1Oa8t7CyvrAL2DY75EaJfECcNqbjEwsuyCw6EjKV23SEc8AnQga7akoIGgZOh51LpePnyaAVN36UuFMT9HF8v3Vg7wms06GtmJjuugCjuaL4UGAg2yLTf5lV7-nHAvPGjJiVOsEMPm-fjM='
    'title' => 'EXAMPLE'
    'applyStencilId' => '1'
    'handle' => 'example'
]
```

### Steps to reproduce

1. Click to create a new form
2. Enter a name and select the "Contact Form" stencil
3. Click Next

### Form settings

N/A

### Craft CMS version

Craft Pro 4.1.4.1

### Plugin version

2.0.1

### Multi-site?

No

**Comment by engram-design** [comment]
It's possible your default Stencil might be out of date, missing the `recipients` value, as I can't replicate this on a fresh install, or on any of my tests. Re-saving the stencil should do the trick, but this is also fixed for the next release. To get this early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.1",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [2.0.2](https://github.com/verbb/formie/releases/tag/2.0.2)
